public/assembler_playground.js

```diff
@@ -108,66 +100,241 @@ class EmulatedInstruction {
     mnemonic = undefined;
     instructionAssembly = undefined;
     instructionOperations = [];
+    constructor (mnemonic, instructionAssembly, instructionOperations) {
+        this.mnemonic = mnemonic;
+        this.instructionAssembly = instructionAssembly;
+        this.instructionOperations = instructionOperations;
+    }
+}
+
+//? Format Definitions
+// Operands
+const operandA = new InstructionOperand("A", "Dest-Register", undefined, 11, 8);
```

Comment:
Make things like this an enum:
"Dest-Register"

when ever a string is known and not dynamic we use an enum(you can easily load an enum from json)